### PR TITLE
{chem}[GCC/12.2.0,intel-compilers/2022.2.1] libxc v6.1.0

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/27b23fa2b4854a3e8a7bd9a5528ae888 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

File: easybuild/easyconfigs/l/libxc/libxc-6.1.0-GCC-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+source_urls = ['https://www.tddft.org/programs/libxc/down.php?file=%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['a3aa16915942543031a5d9c4a92c439ce54249bdcda8c91c4e69e65329dc9a54']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Perl', '5.36.0'),
+]
+
+separate_build_dir = True
```
Comment (boegel) [inline easybuild/easyconfigs/l/libxc/libxc-6.1.0-GCC-12.2.0.eb:21]: @Micket This is enabled by default in `CMakeMake`, so can be removed

File: easybuild/easyconfigs/l/libxc/libxc-6.1.0-GCC-12.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+parallel = 1
+
+# make sure that built libraries (libxc*.so*) in build directory are picked when running tests
+# this is required when RPATH linking is used
+pretestopts = "export LD_LIBRARY_PATH=%(builddir)s/easybuild_obj:$LD_LIBRARY_PATH && "
+
+runtest = 'test'
+
+sanity_check_paths = {
+    'files': ['bin/xc-info'] +
```
Comment (boegel) [inline easybuild/easyconfigs/l/libxc/libxc-6.1.0-GCC-12.2.0.eb:40]: Is it worth adding a sanity check command for `xc-info`?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3101.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/bfe945e4c2deef87adff18f446f5edf2 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c1 - Linux Rocky Linux 8.6, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/1d5a4d8960b44ed65ae516deef298a39 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bwd-rockylinux-91 - Linux Rocky Linux 9.1 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 525.105.17, Python 3.9.14
See https://gist.github.com/SebastianAchilles/dd682c6ec8aca50ee5d10ee880050250 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/2648dc438eca190683c2338fb0b666d5 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @Micket!

**Pull request merged**